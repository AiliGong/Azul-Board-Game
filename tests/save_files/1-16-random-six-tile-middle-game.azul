#myformat
3
6
0
a
b
RRBUBLBUYLBROOUROLRLOYBUROBUOBYYRLRRYLYYBUUUYYUYLOBLOYLLRURLYRLUYOUBRBORBBBLLUYOYYLOBUUOLRRYUBLYYOLBOUOBUBOYROOLOBLURRUR
turn 1 R 2
turn 2 B 2
turn 3 L 1
turn 4 O 3
turn 5 L 3
turn 0 U 5
turn 0 L 3
turn 0 R 4
turn 0 B 4
turn 0 O 3
turn 0 Y 5
turn 1 U 5
turn 2 R 1
turn 3 Y 2
