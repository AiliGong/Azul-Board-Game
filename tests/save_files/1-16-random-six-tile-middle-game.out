Factories:
0: F O Y B O B U O B
1:
2:
3:
4: R L R R
5: Y L Y Y

Score for a: 3
Mosaic for a:
1:           R || . . . . L . 
2:         . . || . . . R . . 
3:       . . . || L . . . . . 
4:     . . B B || . . . . . . 
5:   . . . . Y || . . . . . . 
6: . . . . . . || . . . . . . 
broken:

Score for b: 2
Mosaic for b:
1:           . || . . . . . . 
2:         Y Y || . B . . . . 
3:       . . . || . O . . . . 
4:     . R R R || . . . . . . 
5:   . U U U U || . . . . . . 
6: . . . . . . || . . . . . . 
broken:
